api/src/main/resources/rdr.yaml

```diff
@@ -237,6 +240,22 @@ definitions:
           - ACTIVE
           - INACTIVE
 
+  RdrWorkspaceCreator:
+    type: object
+    description: Details for workspace creator
+    required:
+      - userId
+      - givenName
+      - familyName
+    properties:
+      userId:
+        type: integer
```

Comment:
please add a description about this ID. Like is it for our database or some other thing?